yarn.lock

```diff
@@ -89,7 +89,7 @@
   dependencies:
     "@babel/highlight" "^7.0.0"
 
-"@babel/core@^7.1.0", "@babel/core@^7.6.3", "@babel/core@^7.7.0":
+"@babel/core@^7.1.0", "@babel/core@^7.7.0":
```

Comment:
why is `@babel/core@^7.1.0` in here?

Reply:
Because of jest-config@^24.9.0:
https://github.com/xing/hops/blob/upgrade-untool/yarn.lock#L7675